processing/src/main/java/org/apache/druid/segment/UnnestStorageAdapter.java

```diff
@@ -34,6 +34,7 @@
 import org.apache.druid.segment.column.ValueType;
 import org.apache.druid.segment.data.Indexed;
 import org.apache.druid.segment.data.ListIndexed;
+import org.apache.druid.segment.filter.AndFilter;
```

Comment:
I believe this would be an unused import

Reply:
fixed